docs/troubleshoot/troubleshoot-apiml.md

```diff
@@ -547,3 +547,57 @@ If you are using Zowe's local CA certificate and you still receive **ZWED0148E**
 ```
 
 In this case, ensure that the label names exactly match the names in TSO when confirming your keyring. Any difference in spaces, capitalization, or other places throw the error.
+
+### ZWEAM400E Error initializing SSL Context when using Java 11
+
+**Symptom:**
+
+API ML components configured with SAF keyring are not able to start due to an unrecoverable exception. The message indicates that `safkeyring` is unknown protocol. 
+
+**Examples:**
+```
+2023-06-27 13:07:04.493 <ZWEAGW1:main:394418> ZWESVUSR ERROR (o.z.a.s.HttpsFactory) error java.net.MalformedURLException: unknown protocol: safkeyring                                           
+.at java.base/java.net.URL.<init>(URL.java:652)                                                        
+.at java.base/java.net.URL.<init>(URL.java:541)                                                        
+.at java.base/java.net.URL.<init>(URL.java:488)                                                        
+.at org.zowe.apiml.security.SecurityUtils.keyRingUrl(SecurityUtils.java:246)
+...                                                           
+```
+```
+2023-06-27 13:07:04.528 <ZWEAGW1:main:394418> ZWESVUSR ERROR (o.z.a.s.HttpsFactory) ZWEAM400E Error initializing SSL Context: 'unknown protocol: safkeyring' 
+```
+
+**Solution:**
+
+Starting with Java 11, the safkeyring URLs are now dependent on the type of RACF keystore as seen below.
+ - The URL for a JCECCARACFKS keystore is now `safkeyringjcecca://ZWESVUSR/ZOWERING` 
+ - The URL for a JCERACFKS keystore is now `safkeyringjce://ZWESVUSR/ZOWERING` 
+ - The URL for a JCEHYBRIDRACFKS keystore is now `safkeyringjcehybrid://ZWESVUSR/ZOWERING`
+
+### Failed to load JCERACFKS keyring when using Java 11
+
+**Symptom:**
+
+API ML components do not start properly because they fail to load the JCERACFKS keyring. The exception message indicates that the keyring is not available. 
```

Comment:
This does not seem to be a symptom, but rather an explanation of a symptom. How does the user know that the JCERACFKS keyring fails to load? It there a message thrown? Is that the example?

Reply:
Well the symptom is that all the API ML components do not start correctly. So API ML and whole zowe is unusable.

Reply:
ok. It's clear.